src/components/Products/products.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import ProductCard from "./productCard";
+
+const Products = ({products = [], searchTerm}) => {
+    return (
+        <>
+            {
+                products.length > 0
+                ? products.map((product, i) => <ProductCard key={`${product.data.id || product.id}-card-${i}`} data={product.data} id={product.id} />)
```

Comment:
Are you using [Prettier](https://prettier.io/)? This helps us to review the code. But it is optional.